tests/simpleTests.R

```diff
@@ -34,7 +34,7 @@ as.POSIXct(x)
 as.POSIXct(x+1000)
 as.POSIXlt(x)
 as.POSIXlt(x+1000)
-as.Date(x)
+as.Date(x, tz="UTC")
```

Comment:
The only thing I am wondering is ... that we are loosing an automatic, implied `as.Date(x)` where the UTC timezone is given.